MDANSE/Tests/UnitTests/Analysis/test_scattering.py

```diff
@@ -52,8 +52,8 @@ def dcsf():
     parameters["instrument_resolution"] = ("Ideal", {})
     parameters["output_files"] = (temp_name, ("MDAFormat",), "INFO")
     parameters["q_vectors"] = (
-        "SphericalLatticeQVectors",
-        {"seed": 0, "shells": (5.0, 36, 10.0), "n_vectors": 10, "width": 9.0},
+        "GridQVectors",
```

Comment:
Why the change to `GridQ`?

Reply:
Do we need seed fixing? Not for this PR, just in general?

Reply:
You can fix the seed, but I think you will get different q vectors dependent on the machine or library versions, etc which would cause the tests to fail.